scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -148,3 +148,40 @@ endif::openshift-enterprise[]
 If you add a new node to or remove an existing node from a Cassandra cluster,
 the data stored in the cluster rebalances across the cluster.
 ====
+
+ifdef::openshift-enterprise[]
+To scale down:
+
+. Run the following for the Cassandra node you wish to remove:
++
+----
+$ oc exec <pod> [-c <container>]
```

Comment:
this should either be 'oc rsh <pod> [-c <container>]' or 'oc exec -it <pod> [-c <container>] nodetool decommission'

Since we only have 1 container in our pods, we don't actually need the [-c <container>] part here

Reply:
yes, two ways
1.
# oc rsh ${hawkular-cassandra-pod}
# nodetool decommission
2.
# oc exec -it ${hawkular-cassandra-pod} nodetool decommission